server.js

```diff
@@ -8,23 +8,24 @@ const app = require('app.js');
 const logger = require('logger');
 const https = require('https');
 
-const fs = require('graceful-fs');
 const webpack = require('webpack');
 const webpackDevConfig = require('./webpack/webpack.dev.js');
 const webpackMiddleware = require('webpack-dev-middleware');
+const fs = require('fs');
+const mountSecrets = require('./services/mountSecrets');
 
 const logPath = 'server.js';
 
 if (process.env.NODE_ENV === 'development') {
   const compiler = webpack(webpackDevConfig);
   const wp = webpackMiddleware(compiler, { publicPath: webpackDevConfig.output.publicPath });
   app.use(wp);
-
+  mountSecrets();
   wp.waitUntilValid(stats => {
     app.locals.webpackHash = stats.hash;
     https.createServer({
-      key: fs.readFileSync('keys/server.key'), // eslint-disable-line
-      cert: fs.readFileSync('keys/server.cert') // eslint-disable-line
+      key: fs.readFileSync('secrets/sscs-aat/server-key'), // eslint-disable-line
+      cert: fs.readFileSync('secrets/sscs-aat/server-certificate') // eslint-disable-line
```

Comment:
do we need these eslint-diable-line tags?

Reply:
Removing will cause an eslint error due to the no sync rule (so the method would need to be rewritten to use readfile instead).